packages/config/config/devices/0x0176/tzwp-102.json

```diff
@@ -93,6 +94,81 @@
 					"value": 0
 				}
 			]
+		},
+		{
+			"#": "17",
+			"label": "Power Consumption Reports: Relative Change Threshold",
+			"valueSize": 1,
+			"unit": "%",
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 10,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "101",
+			"label": "Power (W) report interval",
+			"valueSize": 4,
+			"unit": "seconds",
+			"minValue": 5,
```

Comment:
Oh... one more thing I missed. `minValue` needs to be 0. We generally explain the gap in the valid range with a description, e.g. "Allowable range: 5-2678400"

Same for the following parameters

Reply:
Gotcha, was wondering about that one. If it accepted an array of [0, 5-n] or something.
Fixing coming up.